xds/internal/balancer/edsbalancer/eds_impl.go

```diff
@@ -145,42 +146,24 @@ func (edsImpl *edsBalancerImpl) HandleChildPolicy(name string, config json.RawMe
 // updateDrops compares new drop policies with the old. If they are different,
 // it updates the drop policies and send ClientConn an updated picker.
 func (edsImpl *edsBalancerImpl) updateDrops(dropPolicies []xdsclient.OverloadDropConfig) {
```

Comment:
Nit: Should the input parameter be renamed to `dropConfig` instead?

Reply:
Done